packages/web-components/src/components/cbp-banner/cbp-banner.scss

```diff
@@ -0,0 +1,31 @@
+
+:root {
+  --cbp-banner-color-bg: var(--cbp-color-info-dark);
+  --cbp-banner-color-fg: var(--cbp-color-text-lightest)
```

Comment:
I name this like CSS in the sense that I just use "color" to mean the text/foreground color, so drop the "-fg". You'll also want to add a "-dark" version for each of them in `:root` when a dark mode is supported. Is this design the same in light and dark modes?